src/main/resources/assets/paths/tdrDraftDAR.yaml

```diff
@@ -1,6 +1,9 @@
 post:
-  summary: createDraftDataAccessRequest
-  description: Create Draft Data Access Request
+  deprecated: true
```

Comment:
Did we want to deprecate this now and not wait for the cleanup to completely switch over to the endpoints that enforce DAAs?

Reply:
I would prefer this since we need to leave these older APIs in place for at least a release